developing/languages/README.md

```diff
@@ -8,7 +8,7 @@ While Truss may have opinions on particular languages and strategic reasons for
 
 ## Languages Trussels are proficient in
 
-- [Go] (./GO.md) 
+- Go
```

Comment:
Some of the resources in the Go page was already in the languages page. Once the Go resources expands, then we can make it a separate page. Just thought to tighten things up rather than one page just having 4 links in it.